requirements.txt

```diff
@@ -4,11 +4,9 @@ decorator==4.3.0
 docutils==0.14
 ebaysdk==2.1.5
 feedparser==5.2.1
-gevent==1.1.2 ; sys_platform != 'win32' and python_version < '3.7'
+gevent==1.3.7 ; python_version < '3.7'
 gevent==1.5.0 ; python_version >= '3.7'
-gevent==1.4.0 ; sys_platform == 'win32'
-greenlet==0.4.10 ; python_version < '3.7'
-greenlet==0.4.15 ; python_version >= '3.7'
+greenlet==0.4.15
```

Comment:
According to
* https://github.com/gevent/gevent/issues/1685
* https://github.com/gevent/gevent/issues/1680
shouldn't be this greenlet 0.4.17 and gevent 20.9.0 for python 3.7?